src/lib/operators/projection.cpp

```diff
@@ -112,6 +112,7 @@ std::shared_ptr<const Table> Projection::_on_execute() {
   jobs.reserve(chunk_count);
 
   const auto expression_count = expressions.size();
+  const auto forwardable_pqp_columns = _determine_forward_columns(output_table_type);
```

Comment:
Naming: Isn't the whole point of this PR that we don't forward all columns that are forwardable?

Reply:
Do you think changing to `forward_pqp_columns` is sufficient?

Reply:
forwarded?